docs/gambit/gambit.md

```diff
@@ -1,190 +1,328 @@
+which you can manually place on your path or invoke directly (e.g., by calling
+`path/to/gambit/target/release/gambit`).
 
 ## Usage
-- If you installed Gambit using `cargo install --path .` described above,
-  you can learn how to use Gambit by running `gambit mutate -h`.
-- If you went for a local build, you can run `cargo gambit-help` for help.
-- You can print log messages by setting the environment variable
-  `RUST_LOG` (e.g., `RUST_LOG=info cargo gambit ...`).
-  This will show colored diffs of the mutants on your standard output.
-
-`cargo gambit-help` will show you the following message
-  that lists all the command line arguments that Gambit accepts.
-Some of the simple arguments are `num-mutants (default 5)`
-  which lets you control the number of mutants you want to generate,
-  the `seed (default 0)` that controls
-  the randomization of the generated mutants,
-  and `outdir (default out)` that lets you choose
-  where you want to output the mutant files.
 
+Gambit has two main commands: `mutate` and `summary`. `gambit mutate` is
+responsible for mutating code, and `gambit summary` is a convenience command for
+summarizing generated mutants in a human-readable way.
+
+Running `gambit mutate` will invoke the solidity compiler via `solc`, so make
+sure it is visible on your `PATH`. Alternatively, you can specify where Gambit can
+find the Solidity compiler with the option `--solc path/to/solc`, or specify a
+version of solc (e.g., solc8.12) with the option `--solc solc8.12`.
+
+### Running  `gambit mutate` 
+
+The `gambit mutate` command expects either a `--filename` argument or a `--json`
+argument.  Using `--filename` allows you to specify a specific Solidity file to
+mutate:
+
+```bash
+gambit mutate --filename file.sol
 ```
-Command line arguments for running Gambit. Following are the main ways to run it.
 
-1. cargo gambit path/to/file.sol: this will apply all mutations to file.sol.
+However, if you want to mutate multiple files or apply a more complex set of
+parameters, we recommend using a configuration file via the `--json` option
+instead:
 
-2. cargo run --release -- mutate -f path/to/file1.sol -f path/to/file2.sol: this will apply all mutations to file1.sol and file2.sol.
+```bash
+gambit mutate --json gambit-conf.json
+``` 
 
-3. cargo gambit-cfg path/to/config.json: this gives the user finer control on what functions in which files, contracts to mutate using which types of mutations.
+Run `gambit --help` for more information.
 
-Usage: gambit mutate [OPTIONS]
+_**Note:** all relative paths specified in a JSON configuration file are interpreted
+to be relative to the config file's parent directory._
 
-Options:
-  -j, --json <JSON>
-          Json file with config
+In the following section we provide examples of how to run Gambit using both
+`--filename` and `--json`. We provide more complete documentation in the
+[Configuration Files](#configuration-files) and [CLI-Options](#cli-options) sections below.
```

Comment:
The correct syntax for myst references is
```
{ref}`ref-name`
```

where you've added a label to the referenced section, like:
```
(ref-name)=
## Referenced section
```